lib/Sema/TypeChecker.h

```diff
@@ -997,10 +1005,25 @@ class TypeChecker final : public LazyResolver {
   /// the upper bound for the number of seconds we'll let the
   /// expression type checker run before considering an expression
   /// "too complex".
+  /// If zero, do not limit the checking.
```

Comment:
Is it worth building this into SwitchCheckingInvocationThreshold too?

Reply:
Yes, it is, but the comment was incomplete. Fixed.